mod/01_rating/set/type/metric_value.rb

```diff
@@ -240,26 +238,27 @@ def legend args
         path_opts: { slot: { show: :menu, optional_horizontal_menu: :hide } }
       )
     ) # ,:html_args=>{:class=>"td year"}))
-    %{
+    <<-HTML
       <span class="metric-value">
         #{modal_link}
       </span>
-    }
+      HTML
   end
 
   view :timeline_data do |args|
-
-    #container elements
+    # container elements
     dot =  content_tag(:div, '', class: 'timeline-dot')
     value_details_toggle = content_tag(
-                          :i,
-                          '',
-                          class:'fa fa-caret-right fa-lg margin-left-10 btn btn-default btn-sm',
-                          data:{
-                            toggle:'collapse-next',
-                            parent:'.value',
-                            collapse:'.metric-value-details'}
-                          )
+                                        :i,
+                                        '',
+                                        class: 'fa fa-caret-right
+                                                fa-lg margin-left-10
+                                                btn btn-default btn-sm',
```

Comment:
I think this means you have a lot of whitespace and line breaks in your class definition.
I would do it like this:
``` 
'string1' \
'string2'
```